app/javascript/components/rooms/UserBoardIcon.jsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+export default function UserBoardIcon() {
+  return (
+    <svg
```

Comment:
:fire: 
We can further leverage this in other areas so it makes more sense to be placed in the `shared_components/icons` directory maybe?

Reply:
I don't think it is necessary until we plan use it somewhere else.